examples/gallery/symbols/points_categorical.py

```diff
@@ -74,9 +74,9 @@
     # Use bill length and bill depth as x and y data input, respectively
     x=df.bill_length_mm,
     y=df.bill_depth_mm,
-    # Vary symbol size according the body mass, scaled by 7.5e-5
+    # Vary symbol size according to the body mass, scaled by 7.5e-5
     size=df.body_mass_g * 7.5e-5,
-    # Points colored by categorical number code (refers to the species)
```

Comment:
I think parenthesis be more appropriate here than a comma, since it's explaining what the categorical number code represents.

Reply:
I have no too big preference here.
This is changed in commit https://github.com/GenericMappingTools/pygmt/pull/2532/commits/b67d3b8c64895d8a2bec2b59686ec2e8795a90c3. I also updated PR #2531 in commit https://github.com/GenericMappingTools/pygmt/pull/2531/commits/e92f1fc18f1f0974786962096a5fdb7088f3d160.